source/expressions.tex

```diff
@@ -143,7 +143,10 @@
 or computes the value of an operand of an operator,
 as specified by the context in which it appears,
 or an expression that has type \cv{}~\tcode{void}.
-\item An \defn{xvalue} is a glvalue that denotes an object or bit-field whose resources can be reused (usually because it is near the end of its lifetime).
+\item An \defn{xvalue} is a glvalue that denotes an object or bit-field and is not an lvalue.
+\begin{note}
+Such expressions typically denote entities near the end of their lifetime whose resources can be reused.
+\end{note}
 \item An \defn{lvalue} is a glvalue that is not an xvalue.
```

Comment:
I don't think this is really a reasonable pair of definitions any more, due to the circularity. I agree the old definition of "xvalue" was also not really a definition, but I'm not convinced this is an improvement. Perhaps we could arrange this more like "A *glvalue* is either an *lvalue* or an *xvalue*; lvalues typically refer to this kind of thing and xvalues refer to that kind of thing", but we should make sure we don't lose the "no function xvalues" property. But I don't have a good suggestion, so I'm leaning towards leaving this as-is, even though the existing wording is muddying semantics with intent.